source/georef/georef.h

```diff
@@ -238,6 +205,9 @@ struct GeoRef {
     typedef flat_enum_map<nt::Mode_e, ProjectionData> ProjectionByMode;
     std::vector<ProjectionByMode> projected_stop_points = {};
 
+    typedef std::unordered_map<nt::GeographicalCoord, ProjectionByMode> ProjectedCoords;
+    ProjectedCoords projected_coords;
```

Comment:
this is not serialized?

Reply:
Good point, but it;s not serialized. The projection pre-compute is performed by `build_proximity_list()` each time we load or re-load data.